cmake/toolchain/gr740-vxworks7.cmake

```diff
@@ -0,0 +1,88 @@
+# Set system name
+set(CMAKE_SYSTEM_NAME VxWorks)
+set(CMAKE_SYSTEM_VERSION 7.0)
+set(CMAKE_SYSTEM_PROCESSOR gr740)
+
+include_directories(SYSTEM "/opt/tools/sabertooth/sabertooth-bsp/LeonVSB/krnl/h/public")
+include_directories(SYSTEM "/opt/tools/sabertooth/sabertooth-bsp/LeonVSB/krnl/h/system")
+include_directories("/opt/tools/sabertooth/sabertooth-bsp/LeonVSB/share/h")
```

Comment:
These lines should use `%(VSB_HOME)s`

Reply:
The `%X` variables are only available when the make is run (`fprime-util build`), not during the cmake generate stage (`fprime-util generate`), so I can't use them.

Reply:
Ah, okay.